SDWebImageFLPlugin/Classes/FLAnimatedImageBridge/SDWebImageFLCoder.m

```diff
@@ -0,0 +1,74 @@
+/*
+ * This file is part of the SDWebImage package.
+ * (c) Olivier Poitrey <rs@dailymotion.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+#import "SDWebImageFLCoder.h"
+#import "FLAnimatedImageView+WebCache.h"
+
+@implementation SDWebImageFLCoder
+
++ (SDWebImageFLCoder *)sharedCoder {
+    static SDWebImageFLCoder *coder;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        coder = [[SDWebImageFLCoder alloc] init];
+    });
+    return coder;
+}
+
+- (BOOL)canDecodeFromData:(NSData *)data {
+    return ([NSData sd_imageFormatForImageData:data] == SDImageFormatGIF);
+}
+
+- (UIImage *)decodedImageWithData:(NSData *)data options:(SDImageCoderOptions *)options {
+    SDWebImageContext *context = options[SDImageCoderWebImageContext];
+    NSString *operationKey = context[SDWebImageContextSetImageOperationKey];
+    
+    // Check if image request come from `FLAnimatedImageView`
+    if ([operationKey isEqualToString:NSStringFromClass(FLAnimatedImageView.class)]) {
```

Comment:
Compare String is not sufficient, `Class` would be better, because user maybe subclass `FLAnimatedImageView`.